src/style.scss

```diff
@@ -0,0 +1,148 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:regular,500,700&display=swap");
+
+body {
+  font-family: "Roboto", sans-serif ;
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  padding: 0 16px;
+  width: 200px;
+  height: 406px;
+  border: 1px solid rgba(243, 243, 243, 1);
+
+  &__icon {
+    display: flex;
+    justify-content: center;
+    margin-top: 32px;
+    margin-bottom: 40px;
+    img {
+      width: 160px;
+      height: 134px;
+    }
+  }
+
+  &__title {
+    display: flex;
+    flex-direction: column;
+    margin: 0 auto;
+    font-family: inherit;
+    color: rgba(6, 11, 53, 1);
+  }
+
+  &__text {
+    text-align: left;
```

Comment:
not fixed:
![image](https://github.com/mate-academy/layout_product-cards/assets/86923089/5ea3ff79-e990-4ae8-a062-a0a500d5ad2f)